src/pyosmeta/models/base.py

```diff
@@ -130,6 +130,19 @@ class PersonModel(BaseModel, UrlValidatorMixin):
     location: Optional[str] = None
     email: Optional[str] = None
 
+    @field_validator(
+        "emeritus_advisory",
+        "advisory",
+        "emeritus_editor",
+        "editorial_board",
+        mode="before",
+        always=True,
```

Comment:
@lwasser It looks like this field is causing an issue with CI. What is the purpose of `always`?

Reply:
oh shoot @willingc i merged this too quickly. in the PR the tests had passed  (well i thought they had maybe they hadn't!) I suspect always=true was made up by LLMS. i often use chatgpt to proof my code / format etc. and it suggested that addition and i used it without researching. this is my mistake. i should know better than to listen to a llm!

Reply:
i looked more closely. one had failed. i missed it. - https://github.com/pyOpenSci/pyosMeta/actions/runs/10201975001/job/28225084163#step:5:28 i'll submit a fix and won't be so rushed this time.

Reply:
Cool! No problem.